cocos/network/SocketIO.cpp

```diff
@@ -35,12 +35,319 @@
 #include <algorithm>
 #include <sstream>
 
+#include "rapidjson/rapidjson.h"
+#include "rapidjson/document.h"
+#include "rapidjson/stringbuffer.h"
+#include "rapidjson/writer.h"
+
```

Comment:
the cocos2d 3rd party libraries has the name json instead of rapidjson...
i think it should be #include "json/xxx.h"